Libraries_API.ipynb

```diff
@@ -713,6 +715,34 @@
     "    -X DELETE"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### /query/&lt;library_id&gt;\n",
+    "Interact with a specific library. The `library_id` (string) is the unique identifier of the library, as specified in the `/libraries` GET response.\n",
+    "\n",
+    "Functions allowed: add query to a library (POST, GET), delete records from a library (POST, GET).\n",
+    "\n",
+    "#### POST\n",
+    "Add or delete a document to a given library. Returns the number of documents added or removed\n",
+    "\n",
+    "Specify that the request is a POST method by adding `-X POST` to your request. You must include a payload, passed via the `-d` flag, in JSON format. Add this header to your request to indicate your payload is in the appropriate format: `-H \"Content-Type: application/json\"`. The payload must include:\n",
+    "  * bibcode: (list) list of bibcodes to be added or removed\n",
+    "  * action: (string) 'add' or 'remove' to add or remove the given bibcodes from the specified library\n",
+    "\n",
+    "*Return data*:\n",
+    "  * number_added: (integer) number of documents added (if 'add' is used)\n",
+    "  * number_removed: (integer) number of documents removed (if 'remove' is used)\n",
+    "\n",
+    "*Permissions*:\n",
+    "The following type of user can add or remove documents:\n",
+    "  * owner\n",
+    "  * admin\n",
+    "  * write"
+   ]
+  },
```

Comment:
Add a code block example here too, to demonstrate usage. Unfortunately it'll have to be a curl - I'm working on a python version but haven't made a PR yet